File: Java/1304_Find_N_Unique_Integers_Sum_up_to_Zero.java
class Solution {
    public int[] sumZero(int n) {
        int[] result = new int[n];
        int s = n/2 * (-1);
        for(int i = 0; i < n; i++){
            result[i] = s+i;
            result[i] += (result[i] >= 0) && (n%2 == 0) ? 1 : 0;
            /*
            result[i] = s+i;
            if(result[i] >=0){
                if(n%2==0){
                    result[i]++;
                }
            }
            */
        }
        return result;
    }
}

File: CS/1304_Find_N_Unique_Integers_Sum_up_to_Zero.cs
public class Solution {
    public int[] SumZero(int n) {
        int[] result = new int[n];
        
        for(int i = 0, v = n/2; i < n; i++, v--){
            if(v==0 && n%2==0)
                v--;
            result[i] = v;
        }
        
        return result;
    }
}
